### DISCUSSION TITLE
IncludeMany 的属性在生成匿名类时丢失

### DISCUSSION THREAD

**Discussion by rocascent** [dotnetcore/FreeSql#1793] [Q&A]
`IncludeMany()`(或者聚合根)在返回匿名类时会丢失查询到的边界内容。比如：

```csharp
var list = repository.Select
    .ToList(a => new { a });
```

因为在多表查询的时候会用到匿名类，想问一下可以多表查询Join的同时查询`OneToMany`或者`ManyToMany`属性吗？

**Comment by 2881099** [comment]
贪婪加载，数据返回，按照文档的方式编写
